### ISSUE TITLE
print function doesn't work after concatenate dataframes

### ISSUE THREAD

**Issue by PBird** [opensource9ja/danfojs#272]
**Describe the bug**
After concatenate dataframes and then if you try to print one of dataframes that concatenated it gives error. 

**To Reproduce**

i took this simple example from documentation 

``` let data = [['K0', 'k0', 'A0', 'B0'], ['k0', 'K1', 'A1', 'B1'],
  ['K1', 'K0', 'A2', 'B2'], ['K2', 'K2', 'A3', 'B3']]

  let data2 = [['K0', 'k0', 'C0', 'D0'], ['K1', 'K0', 'C1', 'D1'],
  ['K1', 'K0', 'C2', 'D2'], ['K2', 'K0', 'C3', 'D3']]

  let colum1 = ['Key1', 'Key2', 'A', 'B']
  let colum2 = ['Key1', 'Key2', 'A', 'D']

  let df1 = new dfd.DataFrame(data, { columns: colum1 })
  let df2 = new dfd.DataFrame(data2, { columns: colum2 })


  let com_df = dfd.concat({ df_list: [df1, df2], axis: 1 })


  df1.print() 


```

**Expected behavior**
print dataframe 

**Screenshots**
![image](https://user-images.githubusercontent.com/9611415/132257457-19713f3f-ddc1-4473-942d-9afa013d701c.png)

**Desktop (please complete the following information):**
 - OS: linux
 - Browser: chrome
 - Version: 0.2.7

**Comment by risenW** [comment]
@PBird Thanks for raising this issue. The problem has been fixed in the new conversion going on here #235 (internal column data update). So this fix will come with the latest update in a couple of weeks.

**Comment by risenW** [comment]
Fixed in latest version v0.3.0
